app/actions/remote/post.ts

```diff
@@ -935,6 +939,15 @@ export const editPost = async (serverUrl: string, postId: string, postMessage: s
                     await operator.batchRecords(fileDeleteModels, 'delete files');
                 }
             }
+
+            const filesChanged = originalFileIds.length !== file_ids.length ||
+                                !originalFileIds.every((id) => file_ids.includes(id)) ||
+                                !file_ids.every((id) => originalFileIds.includes(id));
```

Comment:
This is probably fine for the amount of attachments we usually have, but the complexity is O(n^2).

Using maps should reduce this to O(n).

Reply:
Done. Nice catch.